docs/release-notes/rockcraft-1-9-0.rst

```diff
@@ -0,0 +1,81 @@
+.. _release-1.9.0:
+
+Rockcraft 1.9.0 release notes
+=============================
+
+28 February 2025
+
+Learn about the new features, changes, and fixes introduced in Rockcraft 1.9.0.
+For information about the Rockcraft release cycle, see the
+:ref:`release_policy_and_schedule`.
+
+
+Requirements and compatibility
+------------------------------
+
+To run Rockcraft, a system requires the following minimum hardware and
+installed software. These requirements apply to local hosts as well as VMs and
+container hosts.
+
+
+Minimum hardware requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- AMD64, ARM64, ARMv7-M, RISC-V 64-bit, PowerPC 64-bit little-endian, or S390x
+  processor
+- 2GB RAM
+- 10GB available storage space
+- Internet access for remote software sources and the Snap Store
+
+
+Platform requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+  :header-rows: 1
+  :widths: 1 3 3
+
+  * - Platform
+    - Version
+    - Software requirements
+  * - GNU/Linux
+    - Popular distributions that ship with systemd and are `compatible with
+      snapd <https://snapcraft.io/docs/installing-snapd>`_
+    - systemd
+
+
+What's new
+----------
+
+Rockcraft 1.9.0 brings the following features, integrations, and improvements.
+
+Remote builds
```

Comment:
Since this is an update or feature enhancement, the heading should describe what's different. Otherwise, per the template this means we're introducing remote builds.

Reply:
Since these two enhancements to remote builds are minor (imo), should we do a `Improvements to remote builds` heading, or move these to a new `Minor Features` section?

Reply:
Either works according to the template, but I feel that interface changes are more important than minor.